assets/queries/cloudFormation/api_gateway_v2_stage_not_access_logging_settings/query.rego

```diff
@@ -0,0 +1,19 @@
+package Cx
+
+CxPolicy [ result ] {
+   document := input.document
+   resource = document[i].Resources[name]
+   resource.Type == "AWS::ApiGatewayV2::Stage"
+   
+   properties := resource.Properties
+   exists_access_log := object.get(properties, "AccessLogSettings", "undefined") != "undefined"
+   not exists_access_log
+   
+      result := {
+                "documentId": 		input.document[i].id,
+                "searchKey":        sprintf("Resources.%s.Properties.AccessLogSettings", [name]),
```

Comment:
If there is a result, that means "AcessLogSettings" isn't defined and can't be in the searchKey.
Replace with:
`"searchKey":        sprintf("Resources.%s.Properties", [name]),`